out/notebooks/provider.ts

```diff
@@ -93,6 +129,9 @@ export class CellResultCountStatusBarProvider implements vscode.NotebookCellStat
                     });
                     // earliest and latest time pickers are only supported for SPL2 at the moment
```

Comment:
why are we having same code block at two places?

Reply:
So there are two states - one for before the search was run and job was dispatched and one after, the icons are slightly different between the two. See this comment from a previous review: https://github.com/splunk/vscode-extension-splunk/pull/89/files#r1248078197